HISTORY.rst

```diff
@@ -7,6 +7,7 @@ Release History
 +++++++++++++++++++
 
 - `SASTokenAuth`, `JWTTokenAuth`, `SASTokenAsync`, and `JWTTokenAsync` now takes keyword argument `refresh_window` to override default token refresh timing in constructors.
+- Updated Azure uAMQP C and Azure C Shared Utility dependencies.
```

Comment:
Are those versionned that we could it there? for clarity? Or a date at least when you copy them?

Reply:
Also, any changelog they wrote so we can know what is fixed?

Reply:
good suggestion! added date & commit information